rdrf/rdrf/services/rpc/rpc_commands.py

```diff
@@ -96,6 +58,10 @@ def rpc_load_matched_patient_data(request, patient_id, questionnaire_response_id
     from registry.patients.models import Patient
     from rdrf.models.definition.models import QuestionnaireResponse
     from rdrf.workflows.questionnaires.questionnaires import Questionnaire
+    from django.utils.translation import ugettext as _
+
+    if not (request.user.is_superuser or request.user.is_staff):
```

Comment:
_Applicable to this and the rest of the `rpc_` commands in this file_

Can we please change this check to `security_check_user_patient`?

Reply:
Added security_check_user_patient where relevant.